test/utils/Deployers.sol

```diff
@@ -96,11 +100,12 @@ contract Deployers {
     function deployMintAndApprove2Currencies() internal returns (Currency, Currency) {
         MockERC20[] memory tokens = deployTokens(2, 2 ** 255);
 
-        address[6] memory toApprove = [
+        address[7] memory toApprove = [
             address(swapRouter),
             address(modifyLiquidityRouter),
             address(donateRouter),
             address(takeRouter),
+            address(settleRouter),
```

Comment:
i dont think this is needed - youre not transferring tokens from the settleRouter